Makefile

```diff
@@ -14,3 +17,7 @@ hdf5comp:
 
 %.patch:
 	curl -f -L -O https://src.fedoraproject.org/rpms/hdf5/raw/master/f/$@
+
+hdf5-$(HDF5_VERSION).tar.$(SRC_EXT):
+	rm -f ./*.tar.{gz,bz*,xz}
+	spectool -g $(NAME).spec
```

Comment:
This should be done in `Makefile_packaging.mk` if the specfile is written correctly.